### ISSUE TITLE
Missing Null Value support

### ISSUE THREAD

**Issue by delkopiso** [rmosolgo/graphql-ruby#364]
The GraphQL spec specifies `null` as a possible argument value (https://facebook.github.io/graphql/#sec-Null-Value). But when I pass null into a mutation like so:

```
mutation {
  user_update(id: "1", address: null) {
    id
    address
  }
}
```

I receive this error: `GraphQL::ParseError - Parse error on "null" (error)`

**Comment by rmosolgo** [comment]
Sorry, I haven't quite added that feature yet! It was only added to the spec [2 days ago](https://github.com/facebook/graphql/pull/83). I'll post back on this issue once its supported here!

**Comment by cjoudrey** [comment]
I'm working on a PR to add this. :grinning:

**Comment by rmosolgo** [comment]
Shipped in 1.2.0

**Comment by delkopiso** [comment]
Wonderful! Thank you so much.
